clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -466,7 +466,7 @@ public Node provide() {
         final void fail(long now, Throwable throwable) {
             if (aborted) {
                 if (log.isDebugEnabled()) {
-                    log.debug("{} aborted at {} after {} attempt(s)", this, now, tries,
+                    log.debug("{} aborted at {} after {} attempt(s)", this, now, ++tries,
```

Comment:
Why do we just increment this is debug logging is enabled?

Reply:
Good point, fixed